api/tokens.go

```diff
@@ -54,14 +54,19 @@ func (capi *census3API) getTokens(msg *api.APIdata, ctx *httprouter.HTTPContext)
 	// parse and encode resulting tokens
 	tokens := GetTokensResponse{Tokens: []GetTokensItem{}}
 	for _, tokenData := range rows {
+		metaTokenID := ""
```

Comment:
I'd not restrict this at this point. If tokenData.MetaTokenID is different from 0, then return it. So we assume that if a MetaTokenID is stored in the database it is correct and we must return it.

The check must exist at the moment we register the token, only a list of allowed token types should be able to define this field on the API call.

Maybe in the future there is some other token impl that requires also an additional ID.